### ISSUE TITLE
"Cannot perform that action, most likely because of errors in source code" in Windows Phone target projects

### ISSUE THREAD

**Issue by GrafSeismo** [ulrichb/SerializationInspections#2]
I tried to add the SerializableAttribute to a Exception class and got only the error message : Cannot perform that action, most likely because of errors in source code.

```C#
    public class SuspensionManagerException : Exception
    {
        public SuspensionManagerException()
        {
        }

        public SuspensionManagerException(Exception e)
            : base("SuspensionManager failed", e)
        {

        }
    }
```

**Comment by ulrichb** [comment]
Hmm, I cannot reproduce this.

When I copy your snippet into an existing solution and hit `Alt+Enter | 'Add Serializable attribute'` on the `SuspensionManagerException` class name I get no error message and the `[Serializable]` attribute is added correctly.

Which VS/R# version do you use?

Can you provide a whole repro solution?

**Comment by GrafSeismo** [comment]
https://code.msdn.microsoft.com/windowsapps/Save-Edited-Picture-16d910c4

I am using Visual Studio 2013 Pro + R# 9.1 Ultimate

**Comment by ulrichb** [comment]
@GrafSeismo 
For Windows Phone targets, the `SerializableAttribute` doesn't even exist (see the [version information in the MSDN docs](https://msdn.microsoft.com/en-us/library/system.serializableattribute.aspx)). I just published an updated version (1.1.1) to the R# gallery which doesn't display the warnings if the `SerializableAttribute` is not resolvable in the context of the analyzed type.

Again, many thanks for your report.
